imap_processing/cdf/utils.py

```diff
@@ -76,7 +77,10 @@ def load_cdf(
     """
     # TODO: remove this when cdflib is updated to version >1.3.0
     if "to_datetime" not in kwargs:
-        kwargs["to_datetime"] = True
+        kwargs["to_datetime"] = True  # type: ignore[assignment]
+        # ToDO Change
+        # Incompatible types in assignment
+        # (expression has type "bool", target has type "dict[Any, Any]")
```

Comment:
```suggestion
        kwargs["to_datetime"] = True  # type: ignore[assignment]
```

This one confuses me. Yes, the target (`kwargs`) is of type dict, but here we are setting a value to one of the dictionary's _keys_ 🤔 

So I think it is fine to just keep this inline `ignore` comment.

Reply:
Thanks! That got it to work.

Reply:
@maxinelasp, just for reference, if the concern was about mutating the input keyword args, the spec says that wont happen.
>  If the form “**identifier” is present, it is initialized to a new ordered mapping receiving any excess keyword arguments, defaulting to a new empty mapping of the same type.